### DISCUSSION TITLE
Missing Git tags

### DISCUSSION THREAD

**Discussion by fgreinacher** [jedisct1/libsodium#1228] [Q&A] [answered]
I cannot find Git tags for the [recent versions on NuGet.org](https://www.nuget.org/packages/libsodium#versions-body-tab).
This makes it a little bit hard to do proper supply chain analysis.
Would it be possible to create the missing tags?

Thanks a ton 🙇

**Comment by ektrah** [comment, marked as answer]
Related discussion: https://github.com/jedisct1/libsodium/discussions/1162
